config/daemon/prometheus.md

```diff
@@ -240,7 +240,7 @@ PS C:\> docker service create --replicas 1 --name my-prometheus
 </div><!-- windows -->
 </div><!-- tabs -->
 
-Verify that the Docker target is listed at http://localhost:9090/targets/.
+Verify that the Docker target is listed at http://127.0.0.1:9090/targets/.
```

Comment:
I don't think `localhost` or `127.0.0.1` would make a difference (only difference I could think of is `localhost` potentially resolving to `::1` (IPv6 localhost).